### ISSUE TITLE
Sigmoid before BCEWithLogitsLoss

### ISSUE THREAD

**Issue by Aaron4Fun** [lopuhin/kaggle-imet-2019#3]
Is the sigmoid activation necessary before the output is fed into BCEWithLogitsLoss? Are there any reasons not using it in your code? Thanks in advance.

**Comment by lopuhin** [comment]
This loss expects logits, so sigmoid must not be added here.
